src/pocketmine/scheduler/AsyncTask.php

```diff
@@ -27,7 +27,7 @@
 /**
  * Class used to run async tasks in other threads.
  *
- * WARNING: Do not call PocketMine-MP API methods, or save objects from/on other Threads!!
+ * WARNING: Do not call PocketMine-MP API methods, or save objects (and arrays cotaining objects) from/on other Threads!!
```

Comment:
Okay, but expect to see more pull requests from me that actually contribute CODE not just templates for issues and pull requests.